demo-program/CMakeLists.txt~

```diff
@@ -0,0 +1,15 @@
+file(GLOB myprog_source "*.c" "*.cpp" "*.h")
+
+#add_library(myprog ${myprog_source} )
+
+
+if (CMAKE_SYSTEM_NAME STREQUAL Windows)
+  target_link_libraries(base udebug iphlpapi psapi)
+else()
+  target_link_libraries(base udebug)
+endif()
+
+add_executable(ourprog myprog.cpp)
+target_link_libraries(ourprog cdd dbm hash)
+
+
```

Comment:
this backup file needs to be deleted and  `*~` added to `.gitignore`

Reply:
btw, I cannot fix this PR up because I cannot pull it, and I cannot pull it because it does not seem to exist: https://github.com/ulriknyman/UCDD/tree/demo-program